torchgen/gen.py

```diff
@@ -1406,7 +1417,17 @@ def get_native_function_declarations(
     *,
     grouped_native_functions: Sequence[Union[NativeFunction, NativeFunctionsGroup]],
     backend_indices: Dict[DispatchKey, BackendIndex],
+    native_function_decl_gen: Callable[
```

Comment:
We should've made this an actual default here in order to preserve backwards compatibility since some users may rely on this having 2 arguments, not 3